R/vars_manager.R

```diff
@@ -512,8 +620,9 @@ create_exception_var <- function(e) {
     }
     return(
         list(
-            name="Introspection Error",
+            name=sprintf("Failed to parse '%s'", var_name),
```

Comment:
I added the name of the variable to this error function to make it more clear when it fails.
And the `has_next_page` parameter below I think is necessary for `jupyter_d1` to properly parse it into a `KernelVariable`.
Then I _think_ the error will show up in the package manager? Am I wrong about that?